css/src/assets/styles/sass/style.scss

```diff
@@ -0,0 +1,230 @@
+@import "reset";
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+body {
+  font-family: "museo-sans",sans-serif;
+  background: #f5f5f5;
+  font-size: 16px;
+  padding: 0.5em;
+}
+img {
+  max-width: 100%;
+}
+a {
+  text-decoration: underline;
+}
+.tile {
+  position: relative;
+  -moz-box-shadow: 0 4px 3px 0 rgba(190,190,190,0.5);
+  -webkit-box-shadow: 0 4px 3px 0 rgba(190,190,190,0.5);
+  box-shadow: 0 4px 3px 0 rgba(190,190,190,0.5);
+  width: 400px;
+  max-width: 100%;
+  margin: 1em auto 0;
+  border-radius: 8px;
+  background: white;
+  cursor: pointer;
+
+  #tile__love {
+    width: 40px;
+    height: 40px;
+    background-color: #eeeeee;
+    background-image: url('../images/icon-heart-outline.svg');
+    background-size: 18px 18px;
+    background-position: center center;
+    background-repeat: no-repeat;
+    border-radius: 50%;
+    position: absolute;
+    top: 1.5em;
+    right: 1.5em;
+    z-index: 1;
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 0.5s;
+    -webkit-transition: opacity 0.5s;
+
+    &.active:before,
+    &.active:after {
+      content:'';
+      display:block;
+      position:absolute;
+      top:0; right:0; bottom:0; left:0;
+      border-radius:50%;
+      border:3px solid #ee362e;
+      -webkit-animation: ripple 0.5s ease-out 1;
+      animation: ripple 0.5s ease-out 1;
+    }
+
+    &.active {
+      background-color: #ee362e;
+      background-image: url('../images/icon-heart.svg');
+    }
+  }
+
+  &:hover {
+    #tile__love {
+      visibility: visible;
+      opacity: 1;
+    }
+  }
+  .tile__top {
+    padding: 0.75em;
+    color: #919191;
+    position: relative;
+
+    .tile__add {
+      position: absolute;
+      width: 60px;
+      background: #f5841f;
+      height: 60px;
+      top: 0;
+      left: 0;
+      border-radius: 8px 0 60px 0;
+      border-right: 8px solid white;
+      border-bottom: 8px solid white;
+      padding: 2px;
+    }
+    h2 {
+      color: #6cc18d;
+      font-weight: bold;
+      font-size: 1.2em;
+      border-bottom: 2px solid #6cc18d;
+      display: inline-block;
+      padding-top: 0.5em;
+      padding-bottom: 0.1em;
+    }
+    .tile__time {
+      float: right;
+      padding: 0.7em 0.4em;
+    }
+  }
+  .tile__bottom {
+    position: relative;
+    display: flex;
+    align-items: center;
+    border-top: 1px solid rgba(190,190,190,0.2);
+    padding: 0.75em;
+    color: #999999;
+
+    p.tile__view-more {
```

Comment:
It's generally a good idea to use a little specificity as possible to apply your styles (i.e. – dropping the combination of `p` and `.tile__view-more`). This will allow for more flexibility for your HTML, but also will reduce the likelihood of running into competing levels of specificity.